skll/data/featureset.py

```diff
@@ -359,3 +368,36 @@ def __getitem__(self, value):
             features = (self.vectorizer.inverse_transform(feats)[0] if
                         self.features is not None else {})
             return self.ids[value], label, features
+
+    @staticmethod
+    def from_data_frame(df, name, labels_column=None, vectorizer=None):
+        '''
+        Helper function to create a FeatureSet object from a `pandas.DataFrame`.
+        Will raise an Exception if pandas is not installed in your environment.
+        `FeatureSet` `ids` will be the index on `df`.
+
+        :param df: The pandas.DataFrame object you'd like to use as a feature set.
+        :type df: pandas.DataFrame
+        :param name: The name of this feature set.
+        :type name: str
+        :param labels_column: The name of the column containing the labels (data to predict).
+        :type labels_column: str or None
+        :param vectorizer: Vectorizer that created feature matrix.
+        :type vectorizer: DictVectorizer or FeatureHasher
+        '''
+
+        if not _HAVE_PANDAS:
+            logger.warning(('pandas not installed.  Please install pandas or '
```

Comment:
This should just raise a `ValueError` if they don't have pandas, because that's the only time anyone should use `from_data_frame`.